ironfish/src/testReporter/testReporter.ts

```diff
@@ -53,26 +52,64 @@ export default class TestReporter implements CustomReporter {
     const writeStream = createWriteStream(
       `${this.globalConfig.rootDir}/${this.reporterConfig.outputDirectory}/${testFileName}.csv`,
     )
-    const stream = format({ headers: true })
-    stream.pipe(writeStream)
 
     // filter console log
     const consoleOutputs = testResult.console?.filter((output) => output.type === 'log')
 
+    // annotation headers for influx data
+    let datatypeHeader = '#datatype,string,long,string,string,string,dateTime:RFC3339'
+    let groupHeader = '#group,false,false,true,true,true,false'
+    let defaultHeader = '#default,_result,,,,,'
+    if (consoleOutputs && consoleOutputs[0]) {
+      const entries = consoleOutputs[0].message.split(',')
+      entries.forEach((input) => {
+        const entry = input.split(':')
+        const value = entry[1]
+        if (value) {
+          const trim = value.replace(/[^\d.-]/g, '')
```

Comment:
What case is this handling?

Reply:
This is to extract the digit value from the result. We have util function that output unit in the result, like heap space with mb. This will ignore all the alphabetic string and keep just the numeric result.